docs/static/breaking-changes.asciidoc

```diff
@@ -12,6 +12,21 @@ after installing it via a package (RPM, DEB).
 respectively. Previously it used to install in `/opt` directory. This change was done to make the user experience 
 consistent with other Elastic products. Full directory layout is described in <<dir-layout>>.
 
+**Default Logging Level:** Changed the default log severity level to INFO instead of WARN to match Elasticsearch. Existing logs
+(in core and plugins) were too noisy at INFO level, so we had to audit log messages and switch some of them to DEBUG
+level.
+
+**Index Template:** The index template for 5.0 has been changed to reflect Elasticsearch's mapping changes. Most
+importantly, the subfield for string multi-fields has changed from `.raw` to `.keyword` to match Elasticsearch's default
+behavior. The impact of this change to various user groups is detailed below:
+
+* New Logstash 5.0 and Elasticsearch 5.0 users - subfields use `.keyword` from the outset. In Kibana, you can use
+`field.keyword` to perform aggregations.
+* Existing users with custom templates - most of you won't be impacted because you use a custom template.
+* Existing users with default template - Logstash does not force you to upgrade templates if one already exists. If you
+intend to move to the new template and want to use `.keyword`, you'll have to reindex existing data. Elasticsearch's
+reindexing API can help move your data from using `.raw` subfields to `.keyword`.
```

Comment:
Can we link `reindexing API` to https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-reindex.html